api-server/server/component-passport.js

```diff
@@ -86,9 +86,12 @@ export const devSaveResponseAuthCookies = () => {
 export const devLoginRedirect = () => {
   return (req, res) => {
     // this mirrors the production approach, but without any validation
-    let { returnTo, origin, pathPrefix } = getRedirectParams(req);
+    let { returnTo, origin, pathPrefix } = getRedirectParams(
+      req,
+      params => params
+    );
     returnTo += isRootPath(getRedirectBase(origin, pathPrefix), returnTo)
-      ? '/learn'
+      ? 'learn/'
```

Comment:
I think for consistency (atleast in all of our API code) we shuold prepend the slashes at all places and leave out the trailing slash at all places.

Reply:
Okay, I'm not 100% sure if we need the trailing slash or not (I was a little confused by what I read about it), but I've taken this one off anyway.  The leading slash was already getting added, so that should be fine.